src/kilo.c

```diff
@@ -874,11 +908,12 @@ void initEditor(void) {
     E.dirty = 0;
     E.filename = NULL;
     E.syntax = NULL;
-    if (getWindowSize(STDIN_FILENO,STDOUT_FILENO,
-                      &E.screenrows,&E.screencols) == -1)
-    {
-        perror("Unable to query the screen for size (columns / rows)");
-        exit(1);
+    E.selection_row = 0;
+    E.selection_offset = 0;
+    if (getWindowSize(STDIN_FILENO, STDOUT_FILENO, &E.screenrows,
```

Comment:
Like you said the other day. If he's using FILENO here, I guess we're good to use it everywhere.